legacy_ibc_testing/simapp/test_helpers.go

```diff
@@ -90,6 +90,7 @@ func SignAndDeliver(
 	t *testing.T, txCfg client.TxConfig, app *bam.BaseApp, header tmproto.Header, msgs []sdk.Msg,
 	chainID string, accNums, accSeqs []uint64, expSimPass, expPass bool, priv ...cryptotypes.PrivKey,
 ) (sdk.GasInfo, *sdk.Result, error) {
+	t.Helper()
```

Comment:
My understanding is that this PR should be a true refactor. What do these calls to `t.Helper()` do?

Reply:
the allow the tests to give us more information.

Reply:
https://pkg.go.dev/testing

Reply:
we can make it more strict, by the way, and it would change more code (I would -- contrary to popular belief, this is manual -- #810 is automatic)